presets/4.3/vtx/HDZero_Freestyle.txt

```diff
@@ -0,0 +1,29 @@
+#$ TITLE: HDZero Freestyle
+#$ FIRMWARE_VERSION: 4.2
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: VTX
+#$ STATUS: COMMUNITY
+#$ KEYWORDS:  vtx, vtx table, HDZero, VTX-L, SA 2.1, HDZero Freestyle, HDZero 1 Watt,
+#$ AUTHOR: Joe Momma
+#$ DESCRIPTION: VTx tables for HDZero 1 Watt VTX capable video transmitters.
+#$ DESCRIPTION:
+#$ DESCRIPTION: 
+#$ DESCRIPTION: ----------
+#$ DESCRIPTION: The information provided on this preset is for educational and entertainment purposes only. Betaflight makes no representations as to the safety or legality of the use of any information provided herein. End users assume all responsibility and liability for ensuring they are complying with all relevant laws and regulations. 
+#$ DESCRIPTION: ----------
+#$ DESCRIPTION: Using the VTX tables as provided may be in breach of your local RF laws. It is up to the end user to research and comply with local regulations and in using these presets the user assumes all liability associated with breaching local regulations.
+#$ DISCUSSION:  https://github.com/betaflight/firmware-presets/pull/157
+#$ INCLUDE_DISCLAIMER: misc/disclaimer/en/vtxtable.txt
+
+# vtxtable
+vtxtable bands 6
+vtxtable channels 8
+vtxtable band 1 BOSCAM_A A FACTORY 0 0 0 0 0 0 0 0
+vtxtable band 2 BOSCAM_B B FACTORY 0 0 0 0 0 0 0 0
+vtxtable band 3 BOSCAM_E E FACTORY 0 0 0 0 0 0 0 0
+vtxtable band 4 FATSHARK F FACTORY 0 5760 0 5800 0 0 0 0
+vtxtable band 5 RACEBAND R FACTORY 5658 5695 5732 5769 5806 5843 5880 5917
+vtxtable band 6 IMD6 I CUSTOM 0 0 0 0 0 0 0 0
```

Comment:
Not a big issue, but the IMD6 band can be removed and 'bands' changed to 5.  It isnt needed, unlike A, B, E to maintain the correct VTx table index for SA2.1.